src/viam/sdk/components/base/base.hpp

```diff
@@ -52,9 +38,6 @@ class Base : public Component, public Stoppable {
     friend std::ostream& operator<<(std::ostream& os, const properties& v);
     friend bool operator==(const properties& lhs, const properties& rhs);
 
-    // functions shared across all components
-    static std::shared_ptr<ResourceRegistration> resource_registration();
```

Comment:
BTW, I should add that I am happy to see the registration entry points removed from the base classes. It was a little weird that the base classes were responsible for registering the client subclasses and associated servers types.